web/static/js/components/idea_board.jsx

```diff
@@ -8,7 +8,7 @@ import styles from "./css_modules/idea_board.css"
 const IdeaBoard = props => {
   const { stage } = props
   const categories = ["happy", "sad", "confused"]
-  const showActionItem = stage !== "idea-generation"
+  const showActionItem = !["idea-generation", "voting"].includes(stage)
```

Comment:
also, once we've done away with the bang, let's update the tests for this condition (it only renders when the stage is either "action-items" or "action-item-distribution"